fdbclient/ServerKnobs.cpp

```diff
@@ -318,8 +318,10 @@ void ServerKnobs::initialize(Randomize randomize, ClientKnobs* clientKnobs, IsSi
 	init( TENANT_CACHE_STORAGE_USAGE_TRACE_INTERVAL,             300 );
 	init( CP_FETCH_TENANTS_OVER_STORAGE_QUOTA_INTERVAL,            5 ); if( randomize && BUGGIFY ) CP_FETCH_TENANTS_OVER_STORAGE_QUOTA_INTERVAL = deterministicRandom()->randomInt(1, 10);
 	init( DD_BUILD_EXTRA_TEAMS_OVERRIDE,                          10 ); if( randomize && BUGGIFY ) DD_BUILD_EXTRA_TEAMS_OVERRIDE = 2;
-	init( DD_MAXIMUM_LARGE_TEAMS,                                 20 ); if( randomize && BUGGIFY ) DD_MAXIMUM_LARGE_TEAMS = 0;
-	init( DD_MAXIMUM_LARGE_TEAM_CLEANUP,                       10000 );
+	
+	// Large teams are disabled when SHARD_ENCODE_LOCATION_METADATA is enabled
+	init( DD_MAXIMUM_LARGE_TEAMS,                                100 ); if( randomize && BUGGIFY ) DD_MAXIMUM_LARGE_TEAMS = 0;
```

Comment:
Does it make sense to also test a higher number of large teams (whatever's the max we might expect to use in the future) if `randomize && BUGGIFY`?